arduino_output/OdomRaw.msg

```diff
@@ -0,0 +1,11 @@
+# Message to transport raw odometry information from
+# the Arduino microcontroller to the main ROS system.
+# It is expected that this raw data will be expanded
+# into the full odometry messages used by ROS.
+
+Header header
+float64 x
```

Comment:
y? Did you mean x?
Griswald or Mark, will one of you be adding in the package.xml and CMakeLists.txt to cause the messages to be generated?